### ISSUE TITLE
Platform entities. Garrison Builder.

### ISSUE THREAD

**Issue by Pasiv01** [Gang-Garrison-2/Gang-Garrison-2#195]
Sorry about this post, I'm not up to standard with computer lingo so its difficult to describe the issue. Anyway...

If I place a platform with coordinates, on the left side of the map...

_top:120 and left:90_ 

... I will get a platform which goes up on about a 45 degree angle. Pretty cool I think.

If i then attempt to place the same kind of platform on the opposite side of the map (symmetrically, on the right side), with...

_top:120 and left: -90 **Minus!!!** - in order to ensure that the platform with move in a 225 degree angle._

... The platform will move away on a 145 degree angle and... just keeping moving away, it doesn't come back.

This is odd because, if I do the exact same thing on the left side of the map i.e _set the coordinates as minus -90_, the platform **will** move on a 225 degree angle.

This isn't a big deal, but I wanted to make a bug report.

TLDR
Garrison builder is crazy!
